lib/modules/home/home_router.dart

```diff
@@ -1,5 +1,14 @@
 part of 'home_module.dart';
 
-abstract class HomeRouter extends Router {}
+abstract class HomeRouter extends Router {
+  void pushToSurveyDetail(
+      {required BuildContext context, required SurveyInfo survey});
+}
 
-class HomeRouterImpl extends HomeRouter {}
+class HomeRouterImpl extends HomeRouter {
+  @override
+  void pushToSurveyDetail(
+      {required BuildContext context, required SurveyInfo survey}) {
+    context.navigator.pushNamed(SurveyDetailModule.routePath);
```

Comment:
did we manage to add `survey` to `SurveyDetailModule`? 😄

Reply:
It will be updated when we integrate `SurveyDetailModule` 😁